packages/aws_bedrock/kibana/dashboard/aws_bedrock-14fd745a-d3c1-4ebe-bd25-00b465336cde.json

```
@@ -262,6 +262,100 @@
+                                                    "label": "Average Guardrails API latency",
+                                                    "operationType": "average",
+                                                    "params": {
+                                                        "emptyAsNull": true,
+                                                        "format": {
+                                                            "id": "duration",
+                                                            "params": {
+                                                                "compact": true,
+                                                                "decimals": 0,
+                                                                "fromUnit": "milliseconds",
+                                                                "toUnit": "asMilliseconds"
+                                                            }
+                                                        }
+                                                    },
+                                                    "scale": "ratio",
+                                                    "sourceField": "aws_bedrock.guardrails.invocation_latency"
+                                                }
+                                            },
+                                            "ignoreGlobalFilters": false,
+                                            "incompleteColumns": {},
+                                            "indexPatternId": "metrics-*",
+                                            "sampling": 1
+                                        }
+                                    }
+                                },
+                                "indexpattern": {
+                                    "layers": {}
+                                },
+                                "textBased": {
+                                    "layers": {}
+                                }
+                            },
+                            "filters": [],
+                            "internalReferences": [],
+                            "query": {
+                                "language": "kuery",
+                                "query": ""
+                            },
+                            "visualization": {
+                                "color": "#CA8EAE",
+                                "layerId": "9bedb80c-4bd2-4206-a1dc-585943b8988a",
+                                "layerType": "data",
+                                "metricAccessor": "2265a51d-c985-462e-9055-3d5008c80b36"
+                            }
+                        },
+                        "title": "",
+                        "type": "lens",
+                        "visualizationType": "lnsMetric"
+                    },
+                    "description": "The latency of the invocations",
```

grammatical mistake:
```suggestion
                    "description": "The latencies of the invocations",
```

---

In the context of the dashboard title, I guess the "The latency of ..." would be the right usage